packages/compiler-sfc/src/script/resolveType.ts

```diff
@@ -294,7 +294,7 @@ function mergeElements(
       }
     }
     if (calls) {
-      ;(res.calls || (res.calls = [])).push(...calls)
+      ;(res.calls ||= []).push(...calls)
```

Comment:
Logical OR is only supported in Node.js 15+. Although Node 14 is EOL, we should not bring in syntax that changes minimum runtime requirements in PRs that are bug fixes. A fix should be focused only on the fix.

Maybe we will do a explicit minimum Node.js version bump in a minor and then we can start using newer syntax. Also do be aware compiler code may need to run in the browser as well.
```suggestion
      ;(res.calls || (res.calls = [])).push(...calls)
```